graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -180,7 +216,7 @@ public String getFingerprint() {
 
             for (final StreamRule streamRule : stream.getStreamRules()) {
                 try {
-                    final Rule rule = new Rule(stream, streamRule);
+                    final Rule rule = new Rule(stream, streamRule, null);
```

Comment:
> TBH, I'd prefer simply using `false` here and making `Rule#isSufficient()` return a primitive `boolean`.

Agreed, avoids null checks as well. Or is there a specific reason why you use `Boolean` instead of `boolean`?